crates/fuel-core/src/database.rs

```diff
@@ -177,19 +177,16 @@ impl Database {
         }
     }
 
-    // TODO: Get `K` and `V` by reference to force compilation error for the current
-    //  code(we have many `Copy`).
-    //  https://github.com/FuelLabs/fuel-core/issues/622
-    fn insert<K: AsRef<[u8]>, V: Serialize, R: DeserializeOwned>(
+    fn _insert<K: AsRef<[u8]>, V: Serialize, R: DeserializeOwned>(
```

Comment:
This style of naming is a little hard to understand. Maybe instead of `_` we should use `raw_`. So `raw_insert` etc.?

Reply:
Usually underscores are meant to denote unused code. Since these fn's are already private idk why we need to underscore them?

Reply:
I thought that we agreed on it [here](https://github.com/FuelLabs/fuel-core/issues/622#issuecomment-1247361991). But it is not true, I didn't get any feedback, I don't know why I thought that=D Reverted it back=)

Reply:
Sorry I didn't see that issue :)